src/widgets/FormatText.js

```diff
@@ -100,7 +103,6 @@ export default class FormatText extends createjs.Container {
         break;
 
       case ROLES.FORMAT_TEXT_TIME:
-        const date = new Date();
```

Comment:
Instead of moving date to the file's global scope, how about adding `{}` around the body of this case statement to keep all the relevant code together?

Reply:
Done